test/json_unittest.cc

```diff
@@ -0,0 +1,385 @@
+#include <functional>
```

Comment:
Let's have this after the first two includes (for consistency).

Reply:
Done.

Reply:
Let's add a blank line before this one.

Reply:
Done.